test/powershell/Modules/Microsoft.PowerShell.Core/Get-Module.Tests.ps1

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
A much needed file!

Reply:
Hopefully, more tests can be added gradually.